build/azuredevops-vars.yml

```diff
@@ -2,11 +2,11 @@
 # by the build
 variables:
   - name: TaskctlVersion
-    value: 1.5.1
+    value: 2.0.5
```

Comment:
Ahh `2.0.5` has a problem in that it doesn't report exit codes correctly. `1.7.5` is the latest taskctl to behave right...

Reply:
```suggestion
    value: 1.7.5
```

Reply:
Let's get a new PR in after this to move to eirctl 🙂